src/StreamJsonRpc.Shared/JsonRpc.cs

```diff
@@ -654,6 +661,38 @@ protected virtual Task<ReturnType> InvokeCoreAsync<ReturnType>(int? id, string t
                 }
             }
         }
+        
+        private static Dictionary<string, List<string>> GetMethodToAttributeMap(object target)
+        {
+            var methodToAttributeMap = new Dictionary<string, List<string>>();
+            if (target != null)
+            {
+                for (TypeInfo t = target.GetType().GetTypeInfo(); t != null; t = t.BaseType?.GetTypeInfo())
+                {
+                    var anyMethodWithAttribute = t.DeclaredMethods.Any(m => m.GetCustomAttribute(typeof(MethodNameAttribute)) != null);
+                    if (anyMethodWithAttribute)
```

Comment:
You're right, I think this was left over from a previous implementation.  I'll remove it.